framework/src/st_things/st_things_request_handler_collection.c

```diff
@@ -569,7 +569,7 @@ static bool handle_get_req_on_collection_batch(things_resource_s *collection_rsr
 		}
 
 		strncat(query, prefix, strlen(prefix));
-		strncat(query, if_type, strlen(if_type));
+		strncat(query, if_type, strlen(if_type) + 1);
```

Comment:
Modificaions to st_things_xx.c files should be discussed and requested to Youngjin Kim in IoT Lab.
Also, strncat automatically add '0' at the end of arg1, so we may not need to modify this.
See "TizenRT/lib/libc/string/lib_strncat.c"

Reply:
modified